next.config.js

```diff
@@ -3,7 +3,18 @@ const nextConfig = {
   reactStrictMode: true,
   typescript: {
     ignoreBuildErrors: true,
-  }
+  },
+  async redirects() {
+    return [
+      {
+        // Redirect / (root of our website) to /roommates
+        // https://nextjs.org/docs/api-reference/next.config.js/redirects
+        source: '/',
+        destination: '/roommates',
+        permanent: false,
+      },
+    ]
+  },
```

Comment:
I think we can omit this redirect for now, because we need some way to view/utilize the index.js file somehow. If we redirect out from the root, index.js becomes useless. We can discuss routing in meeting.

Reply:
sounds good. i did this redirect because I thought the roommate page *was* the front page and index.js is intended to be useless - if we don't want it to be we should definitely discuss